hummingbot/connector/exchange/ascend_ex/ascend_ex_auth.py

```diff
@@ -38,7 +63,7 @@ def get_auth_headers(
         }
 
     @staticmethod
-    def get_headers() -> Dict[str, Any]:
+    def get_meta_headers() -> Dict[str, Any]:
```

Comment:
We should not use the term `meta`. In software development it is heavily associated with objects introspection and metaprogramming